### DISCUSSION TITLE
Can't submit student dev package application

### DISCUSSION THREAD

**Discussion by Ske63427** [community/community#144032] [GitHub Education]
### Hi everyone,

I am trying to get the student dev package, but when i try to apply, i am told i need an email ending with my school board's domain; but i do have this email domain. Github doesn't seem to be able to recognize it, so i can't even submit an application. where do i go from here?

**Comment by codeslayr** [comment]
If everything looks correct on your end but you’re still facing issues, you can reach out to GitHub Support. Provide them with the exact issue, including the email domain you're using and the error message you're receiving. They can manually assist you or help identify why your domain isn't being recognized.
Visit GitHub Support: [GitHub Support](https://support.github.com/)

**Reply by Ske63427** [reply]
I tried that initially, it blocked my ability to write a subject for my ticket and told me to post here(I selected "I have found a GitHub Education technical bug"). Do i just go through each possible subject until i find one that allows be to submit it?
